### PR TITLE
remove the deprecated wiki macros

### PR DESCRIPTION
**Pull request by wilzbach**
WIKI is no longer used

> These tags are back from the days when the official D wiki was http://www.prowiki.org/wiki4d/wiki.cgi.

https://github.com/dlang/dlang.org/pull/1310

For interested: produced by this ugly perl expression

```
for file in **/*.d ; do perl -ne 'push @x, $_;                                                                                           6:25
          if (@x > 2) {
              if ($x[1] =~ /WIKI/) { if ($x[2] =~ /[a-zA-Z\/]/ ) {print $x[2] } undef @x}
                  else { print shift @x }
          }
          } END { print @x' $file > tmp.d && mv tmp.d $file; done
```

(and a manual correction for index.d and std/socketstream.d)

### CODE REVIEW

**Comment by wilzbach** [comment]
> (and a manual correction for index.d and std/socketstream.d)

and std/functional.d and std/getopt.

Ping @CyberShadow and @aG0aep6G

File: std/algorithm/package.d [hunk 1]
```diff
@@ -168,8 +168,6 @@ sort!"a > b"(a);           // predicate as string
 sort(a);                   // no predicate, "a < b" is implicit
 ----
 
-Macros:
```
Comment (aG0aep6G) [inline std/algorithm/package.d]: Must not remove this instance of `Macros:` as there are other macros being defined here. Happened a bunch of times throughout the diff.

**Comment by wilzbach** [comment]
> Must not remove this instance of Macros:

Oh right thanks - but I found that for the container modules `TEXTWITHCOMMAS` is actually not even used. Put this as a separate commit.

File: std/process.d [hunk 2]
```diff
@@ -79,8 +79,6 @@ License:
    $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
 Source:
     $(PHOBOSSRC std/_process.d)
-Macros:
```
Comment (aG0aep6G) [inline std/process.d]: !

File: std/signals.d [hunk 3]
```diff
@@ -45,8 +45,6 @@
  *
  *      Not safe for multiple threads operating on the same signals
  *      or slots.
- * Macros:
```
Comment (aG0aep6G) [inline std/signals.d]: !

File: std/stdint.d [hunk 4]
```diff
@@ -112,8 +112,6 @@
     $(TD the largest unsigned integral type)
     ))
 
- * Macros:
```
Comment (aG0aep6G) [inline std/stdint.d]: !

**Comment by wilzbach** [comment]
Hmm seems like doing everything by hand would have been faster - anyway I found a couple more instances were `Macros` can vanish. Btw after this PR we only have 30 custom `Macros` definitions in Phobos :)
thanks @aG0aep6G for pointing out how stupid I am.

**Comment by CyberShadow** [comment]
There's still a lot of changes in the generated docs.

**Comment by wilzbach** [comment]
> There's still a lot of changes in the generated docs.

Looks a lot better now - the diff now looks like the changes are only due to different HTML tag formatting?

**Comment by CyberShadow** [comment]
Auto-merge toggled on

**Pull request merged**